.githooks/pre-push

```diff
@@ -17,4 +17,4 @@ remote_url="$2"
 
 # ensure that formatting is correct before pushing to the remote repository
 dotnet tool restore
-dotnet fantomas -r src docs/.style/ build.fsx
\ No newline at end of file
+git diff --cached --name-only --diff-filter=ACM -z | xargs -0 dotnet tool run fantomas 
```

Comment:
Don't we need that second line as well:
`git diff --cached --name-only --diff-filter=ACM -z | xargs -0 git add`?